extensions/control-plane/store/sql/transfer-process-store-sql/docs/schema.sql

```diff
@@ -80,3 +80,4 @@ CREATE UNIQUE INDEX IF NOT EXISTS data_request_id_uindex
 CREATE UNIQUE INDEX IF NOT EXISTS lease_lease_id_uindex
     ON edc_lease (lease_id);
 
+ALTER TABLE edc_data_request DROP COLUMN IF EXISTS connector_id;
```

Comment:
this is a schema, not a migration, please remove `connector_id` directly from the schema creation, mark this PR as `breaking-change` and specify what needs to be done to migrate the schema in the text